CONTRIBUTING.md

```diff
@@ -44,7 +44,7 @@ Here are a few guidelines to get started:
 These are all guidelines, not strict rules.
 Use proper judgement, keeping simplicity and user-friendliness as the top priorities.
 
-When in doubt, have a look at a few existing pages :).
+When in doubt, have a look at a few existing pages.
```

Comment:
I have a mixed feeling about it. Although, it does feel informal but at the same time it feels more welcoming in my opinion.

Reply:
You're right — but is that the right place?  Maybe we should adjust wording elsewhere to mimic the feeling.

Reply:
I have mixed opinions, for: its inviting and as a contributor, it removes a takes the edge off contributing to a 33k starred project

against: It seems a bit out of place.